app/src/main/java/com/concordia/mcga/helperClasses/OutdoorPath.java

```diff
@@ -0,0 +1,180 @@
+package com.concordia.mcga.helperClasses;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.akexorcist.googledirection.DirectionCallback;
+import com.akexorcist.googledirection.GoogleDirection;
+import com.akexorcist.googledirection.constant.TransitMode;
+import com.akexorcist.googledirection.constant.TransportMode;
+import com.akexorcist.googledirection.constant.Unit;
+import com.akexorcist.googledirection.model.Direction;
+import com.akexorcist.googledirection.model.Leg;
+import com.akexorcist.googledirection.model.Route;
+import com.akexorcist.googledirection.model.Step;
+import com.akexorcist.googledirection.util.DirectionConverter;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Polyline;
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @Link{OutdoorPath} connects to the Google Directions API in out to find paths between POI
+ * Please refer to http://www.akexorcist.com/2015/12/google-direction-library-for-android-en.html
+ */
+public class OutdoorPath implements DirectionCallback {
```

Comment:
It would make the outdoorDirection class very long since OutdoorPath contains many method. I prefer keeping them in separate file for this reason